.changeset/clever-apes-shop.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/cloudflare': minor
```

Comment:
Should this be a patch? as it fixes a bug
```suggestion
'@astrojs/cloudflare': patch
```

Reply:
Yeah, patch is probably more correct.